charts/backstage/values.yaml

```diff
@@ -297,3 +297,11 @@ serviceAccount:
 
   # -- Auto-mount the service account token in the pod
   automountServiceAccountToken: true
+
+# -- Node tolerations for server scheduling to nodes with taints
+# <br /> Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
+tolerations: []
```

Comment:
These tolerations are meant to be applied to the backstage deployment only, correct? In that case instead of having them as a top level value, I would say we should rather nest them within the `backstage.*` key where the rest of the `Deployment`-specific values are defined (like `backstage.resources`, `backstage.command`, etc.), wdyt?

Reply:
That is great, I am working with Chis on this issue:)

Reply:
Hi!

Thank you for your continued follow-ups.
I was considering where to put it, and since it wasn't directly connected to the backstage application itself, I put it outside like podSecurity.

I'm totally fine with moving it to backstage.* if that is the correct place.

Reply:
Any other opinions on where in the configuration should be?
I'll move it to within backstage.* as suggested for now.

Reply:
@tumido Does it match what you would expect now?

podSecurity was in fact also within the bacstage section, so my argument for keeping it outside didn't make any sense.

Reply:
@chrisaq yes, the layout in this current revision seems logical to me, it's following the same pattern as the rest. :+1: